src/pages/settings/Payments/PaymentMethodList.js

```diff
@@ -60,6 +60,12 @@ const propTypes = {
         walletLinkedAccountType: PropTypes.string,
     }),
 
+    /** Type of active/highlighted payment method */
+    actionPaymentMethodType: PropTypes.oneOf([CONST.PAYMENT_METHODS.DEBIT_CARD, CONST.PAYMENT_METHODS.BANK_ACCOUNT, '']),
```

Comment:
Thanks @parasharrajat for addressing the previous changes! :) 

I found another console error related to the prop `actionPaymentMethodType` when I added a Paypal account:

<img width="500" alt="Screen Shot 2022-01-26 at 10 21 11" src="https://user-images.githubusercontent.com/6829422/151204557-3e97c839-37ed-4068-a130-4b735faeacb0.png">

I was wondering if we can change this line it to something like this (that will keep the methods types up to date if `CONST.PAYMENT_METHODS` changes in the future):

```suggestion
    actionPaymentMethodType: PropTypes.oneOf([..._.values(CONST.PAYMENT_METHODS), '']),
```

Reply:
Good catch. I just noticed my error logging was disabled